courses/static_analysis_suite/150_lab_customization.rst

```diff
@@ -0,0 +1,241 @@
+*********************
+Lab - Customization
+*********************
+
+..
+    Coding language
+
+.. role:: ada(code)
+    :language: Ada
+
+.. role:: C(code)
+    :language: C
+
+.. role:: cpp(code)
+    :language: C++
+
+..
+    Math symbols
+
+.. |rightarrow| replace:: :math:`\rightarrow`
+.. |forall| replace:: :math:`\forall`
+.. |exists| replace:: :math:`\exists`
+.. |equivalent| replace:: :math:`\iff`
+.. |le| replace:: :math:`\le`
+.. |ge| replace:: :math:`\ge`
+.. |lt| replace:: :math:`<`
+.. |gt| replace:: :math:`>`
+
+..
+    Miscellaneous symbols
+
+.. |checkmark| replace:: :math:`\checkmark`
+
+==============
+Lab Setup
+==============
+
+-------------------
+Customization Lab
+-------------------
+
+* Use :toolname:`GNAT Studio` to open the :filename:`default.gpr` project in the :filename:`150_lab_customization` folder
+* Start a baseline run at **level 3**
+* Examine the result
+
+===================
+Filtering Sources
+===================
+
+-----------------------
+Focusing Our Analysis
+-----------------------
+
+The Legacy module is raising the majority of error, and its code is using many dangerous constructs
+
+As a result we want to focus on the module that is not legacy.
+
+* In the locations tab, enter the name of :file:`cruise.adb` in the filter text box
+
+.. image:: codepeer/cruise_filter_in.png
+
+* This shows only messages for the specified file
+
+------------------
+Locations Filter
+------------------
+
+* Open the filter advanced menu by clicking on the magnifying glass icon in the filter bar.
+
+  .. image:: codepeer/cruise_filter_advanced_button.png
+    :width: 50%
+
+  .. image:: codepeer/cruise_filter_advanced_menu.png
+    :width: 50%
+
+  * Notice that you have several options there to filter with advanced messages criterias, as well as a search history.
+
+--------------------
+Filtering Messages
+--------------------
+
+* We want to sort the messages by importance, which is called sub-category in the tool.
+
+  * Click on the "hamburger" menu on the right hand side of the search bar.
+
+.. image:: codepeer/cruise_filter_configuration.png
+
+  * Select "Sort by subcategory"
+
+    * Notice that the messages are now displayed as high category first, then medium.
+
+------------------------
+Filtering Source Files
+------------------------
+
+* Filtering options on existing analysis are useful, but now we want to simply ignore the legacy files when performing the analysis.
+
+  * Open the project file from the :menu:`Project` tab
+
+    *  Right-click on :menu:`Cruise` |rightarrow| :menu:`Project` |rightarrow| :menu:`Edit Source File`.
```

Comment:
The project name is actually `Default `in GNAT Studio, and that is also the name of the gpr file that you open (`default.gpr`). So the text here should read "Right-click on Default -> Project" instead of "Cruise->Project" (unless the project file name is changed)

Reply:
Fixed